apps/backend/app/routers/auth.py

```
@@ -0,0 +1,48 @@
+
+@router.post("/register", response_model=RegisterResponse, status_code=201)
+async def register(body: RegisterRequest) -> RegisterResponse:
+    """Register a new user account."""
+    hashed = hash_password(body.password)
+    try:
+        user = await db.create_user(
+            email=body.email,
+            hashed_password=hashed,
+            display_name=body.display_name,
+        )
+    except (IntegrityError, Exception) as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)